src/modules/Form/components/BirthdateField/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { RefObject } from 'react';
+import { getTodayDate } from 'utils';
+import * as S from './styled';
+
+type Props = {
+  onChange: (errorRef: RefObject<HTMLSpanElement>) => void;
+  refs: {
+    inputRef: RefObject<HTMLInputElement>;
+    errorRef: RefObject<HTMLSpanElement>;
+  };
+};
+
+export const BirthdayField = (props: Props) => {
```

Comment:
```suggestion
export const BirthdayField = ({ onChange, refs }: Props) => {
```
you can also do it there